citation-matching/citation-matching-coansys-code/src/main/java/pl/edu/icm/coansys/citations/coansys/input/RawReferenceToEntityConverterFactory.java

```diff
@@ -33,7 +33,7 @@ public RawReferenceToEntityConverter createRawReferenceToEntityConverter() {
         try {
 
             if (this.model != null) {
-                parser = new CRFBibReferenceParser(this.model);
+                parser = new CRFBibReferenceParser(this.getClass().getResourceAsStream(this.model));
```

Comment:
Now when I think about I believe it would be better for the model to be the method param...